### ISSUE TITLE
DependencyError( psychopy.exceptions.DependencyError: No sound libs could be loaded. Tried: ['pyo']

### ISSUE THREAD

**Issue by monique2208** [psychopy/psychopy#5475]
I encountered this dependency error using Psychopy 2023.1.0 but replicated it in 2023.1.1 as well. Psychopy-pyo was installed in the same environment. 
Since I had no problems directly importing pyo, I investigated further. The problem does not seem to be the actual import but what comes after. 

In psychopy\sound\__init__.py, line 95, backend.pyoSndServer is called, but pyoSndServer is not specified in backend_pyo. This raises an error, which is returned as a DependencyError.

**Comment by djar0** [comment]
Did this issue get resolved within 2023.1.1?

**Comment by monique2208** [comment]
No I also tested it there. But it also looks to me like the relevant code did not change here between versions.

sound\__init__.py tries to get the pyoSndServer and audioDriver from the backend, but these are not in the sound\backend_pyo.py
In line 91 to 98 of __init__.py:
```
        elif thisLibName == 'pyo':
            try:
                from . import backend_pyo as backend
                Sound = backend.SoundPyo
                pyoSndServer = backend.pyoSndServer
                audioDriver = backend.audioDriver
            except Exception:
                continue
```

I had a closer look to find the Exception that is caught here: `module 'psychopy.sound.backend_pyo' has no attribute 'pyoSndServer'`

**Comment by djar0** [comment]
Thanks for your quick reply and looking in-depth into the issue. Have you found a work around solution? Or will we just have to wait for the next version and for the error to hopefully be fixed?

**Comment by emptycross** [comment]
I changed the source code and set these variables to None to test it since this seemed to be the default values in the 2022 version. But although it ran I think there were some issues with the audio quality. I did not spend longer on it since the sound was only for the feedback trials which we decided to discard. If possible I would either work with the last 2022 version which does not seem to have the issue, or try a different audio library.
